src/lib/addresses.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Imported methods from: https://github.com/rsksmart/rsk-utils/blob/master/src/addresses.js
+ */
+
+/**
+ * @description Check if a string is an address
+ * @param {String} address
+ * @returns {Boolean}
+ */
+export function isAddress(address) {
+  return /^(0x)?[0-9a-fA-F]{40}$/.test(address)
+}
```

Comment:
viem already has methods for this
[address utilities](https://viem.sh/docs/ethers-migration#address-utilities)